pressbooks/pressbooks-book
# Underline links in info & meta blocks on book home page

This PR fixes https://github.com/pressbooks/pressbooks-book/issues/875

To test:
1. Checkout this branch
2. Add a long description with link to your book and apply a CC license from book info
3. Visit the home page of the book and make sure that links in book info and metadata sections are underlined at rest
![Screenshot from 2021-10-25 14-55-37](https://user-images.githubusercontent.com/13485451/138776682-04d8b844-4535-49cd-8f47-52d39b4fbe6e.png)
